src/main/resources/application.yaml

```diff
@@ -284,6 +284,9 @@ darts:
       sas-endpoint: ${DETS_SAS_URL_ENDPOINT:}
       container-name: darts-migration
       temp-blob-workspace: ${user.home}/dets/tempworkspace
+      delete-timeout: 60
```

Comment:
whats the unit? do we want to make this into a duration?

Reply:
This was copied from Marios PR when I needed the delete code he had added but it appears this was replaced with a different config setting.